app/config/api.js

```diff
@@ -0,0 +1,27 @@
+const Joi = require('joi')
+
+const schema = Joi.object({
+  documentsApi: Joi.object({
+    baseUrl: Joi.string().required()
+  }),
+  responsesApi: Joi.object({
```

Comment:
this service isnt used. causing crash on local build.

Reply:
Removed